exporters/trace/jaeger/env.go

```diff
@@ -15,33 +15,34 @@
 package jaeger // import "go.opentelemetry.io/otel/exporters/trace/jaeger"
 
 import (
-	"errors"
+	"fmt"
 	"os"
-	"strconv"
-	"strings"
-
-	"go.opentelemetry.io/otel"
-	"go.opentelemetry.io/otel/attribute"
 )
 
 // Environment variable names
 const (
-	// The service name.
-	envServiceName = "JAEGER_SERVICE_NAME"
-	// Whether the exporter is disabled or not. (default false).
-	envDisabled = "JAEGER_DISABLED"
-	// A comma separated list of name=value tracer-level tags, which get added to all reported spans.
-	// The value can also refer to an environment variable using the format ${envVarName:defaultValue}.
-	envTags = "JAEGER_TAGS"
+	// Hostname for the Jaeger agent, part of address where exporter sends spans
+	// i.e.	"localhost"
+	envAgentHost = "OTEL_EXPORTER_JAEGER_AGENT_HOST"
+	// Port for the Jaeger agent, part of address where exporter sends spans
+	// i.e. 6832
+	envAgentPort = "OTEL_EXPORTER_JAEGER_AGENT_PORT"
 	// The HTTP endpoint for sending spans directly to a collector,
 	// i.e. http://jaeger-collector:14268/api/traces.
-	envEndpoint = "JAEGER_ENDPOINT"
+	envEndpoint = "OTEL_EXPORTER_JAEGER_ENDPOINT"
 	// Username to send as part of "Basic" authentication to the collector endpoint.
-	envUser = "JAEGER_USER"
+	envUser = "OTEL_EXPORTER_JAEGER_USER"
 	// Password to send as part of "Basic" authentication to the collector endpoint.
-	envPassword = "JAEGER_PASSWORD"
+	envPassword = "OTEL_EXPORTER_JAEGER_PASSWORD"
 )
 
+// AgentEndpointFromEnv uses environment variables to return the Jaeger agent hostport
+func AgentEndpointFromEnv(e *string) {
```

Comment:
This doesn't look like it needs to be exported and we should prefer to use the return value of a function instead of modifying a passed value reference.

Additionally, this will not return valid values if only the host or port is specified with an environment variable.

```suggestion
// agentEndpointFromEnv uses environment variables to return the Jaeger agent endpoint.
func agentEndpointFromEnv(e string) (string, int) {
```

The return values here could be the host and port.